ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+
+Additionally, many services are dependent on other services, which have their own Product Owners. Therefore, this persona also plays a critical role in managing those dependencies, liaising with other Product Owners and Engineering resources to effectively prioritize work. Using agile concepts, this role is analogous to a 'Product Owner'.
+
+**Engineering Manager**
+
+Work needs to be done. The majority of this work is undertaken by Developers, who comprise teams led by an Engineering Manager. The primary role of this persona is to enable those teams to do the (best) work they need to do, as it is prioritized by the Product Owner. This role must also maintain working relationships with peer Engineering managers, especially where co-dependencies, shared goals and natural affinities exist.
+
+**Developers**
+
+These folks are the 'do-ers' . This persona implements the service in accordance with the 'plan' (as mostly determined by the Product Owner). They turn this plan into reality, creating and maintaining the code which underpins the service. Not only are they responsible for implementing the functionality of the service (in this context "feature work"), but they[^1] are also responsible for how the service 'operates'. Therefore, if reliability is an issue (often identified by the SREs) then they are also responsible for fixing those problems also (called "reliability work").
+
+Also, it is important to acknowledge that there are developers who are also functioning as SREs, and in fact this is definitely in the spirit of Site Reliability Engineering in general. An example may be a smaller Engineering team who fill many roles, or maybe a newer service that is early in its stages of maturity. However, for the purpose of these personas, consider the person acting in the role of a "Developer" when performing that function, and similarly acting in the role of an SRE when performing that function.
+
+**Engineering/Architecture Lead**
+
+The Developers and SREs do not work in isolation. To obtain the economies of scale that are necessary, and to make sure that work is not duplicated, they need to conform to the 'systems (including processes)' and 'architecture' that best suits the business goal. This is the primary role of this persona. They provide the "guard-rails" for the developers, and help keep the services aligned with accepted strategic technical direction. They also have a key role in developing best practices, and coaching teams to avoid anti-patterns.
+
+**Quality Engineer**
+
+Service development has a life-cycle that ultimately results in a service which is deployed and used by customers. The quality phase of that life-cycle is absolutely critical in the reliability of that service, as SREs need to be able to trust what is being deployed. That is the role for this persona. It is important to note, that this is not the role of the SRE, especially testing in Production environments.
```

Comment:
Maybe I'm missing it in the wording, but I don't see a role defined here.
Is the role to 'ensure SRE trust what is deployed' and the place to do that is in the 'quality phase of that (Service development) life-cycle`?

Reply:
Main distinction I see is that QE installs and manages instances of the product for the sole purpose of bug and regression testing, while SRE is operating an instance of the product on behalf of a customer, with bug and regression discovery as an ancillary... benefit... :p  So maybe:

> QE installs and manages instances of the product for the sole purpose of bug and regression testing, informing Developers of reliability issues and grounding confidence for other personas that the product functions as intended in the subset of situations covered by QE.

or something like that?

Reply:
I like what @wking has added. Does that answer your question @david-martin ?

Reply:
Modified as per @wking 's suggestion.